data/menu.js

```diff
@@ -64,7 +64,7 @@ export const menuItems = [
       {
         id: 31,
         label: "Penetration Testing & Vulnerability Assessment",
-        link: "/project-details/Penetration Test of Mach7t",
+        link: "/project-details/Penetration-Test-of-Mach7t",
```

Comment:
Why are we adding hyphens here when `replace` method is handing hyphens automatically?

Reply:
<img width="530" alt="image" src="https://github.com/user-attachments/assets/469b2173-9e02-4312-bdd9-7d8327ee14e3">
Dashes should not show up in breadcrumbs

Reply:
Done.